src/containers/App/Content.tsx

```diff
@@ -219,6 +228,11 @@ interface ContentWrapperProps {
 
 function ContentWrapper(props: ContentWrapperProps) {
     const {singleClusterMode, isAuthenticated} = props;
+    const isForbidded = useAccessTotallyRestricted();
+
+    if (isForbidded) {
+        return <AccessDenied />;
+    }
```

Comment:
As I understand from issue, when `UseLoginProvider` is `false`, you should never show login error, even if `whoami` returns 401. 

In you solution, `UseLoginProvider: false` means, that the cluster will be always under `AcessDenied`

Reply:
100%!